src/components/itemsList.js

```diff
@@ -0,0 +1,16 @@
+ 
+ function ItemsList (props){
+    
+    return (
```

Comment:
- [ ] Please change the path for the component here.
![image](https://user-images.githubusercontent.com/74744787/186278639-64692498-f797-4863-9633-61604ba3da95.png)

Reply:
please can you re-clone it , it works better on my local machine